src/lucky/server_settings.cr

```diff
@@ -4,7 +4,7 @@ module Lucky::ServerSettings
   extend self
 
   def host : String
-    settings["host"].as_s
+    ENV["HOST"]?.try(&.to_s) || settings["host"].as_s
```

Comment:
I'd have to try this out first to make it work as expected. Could you make this a separate PR with some details about what the use case is and how you're using it? I think I have a rough idea, but I always like to check before making changes to default config